parley/src/layout/line_break.rs

```diff
@@ -290,7 +295,11 @@ impl<'a, B: Brush> BreakLines<'a, B> {
                         let boundary = cluster.info().boundary();
                         let style = &self.layout.data.styles[cluster.data.style_index as usize];
```

Comment:
We needn't lookup the style on a per cluster basis IIUC. We [push runs for a given style](https://github.com/linebender/parley/blob/50f480a15bd7bec539e44b945fd6110fdbdc5cfe/parley/src/shape/mod.rs#L398).

No action required - just thought I'd highlight this for potential discussion.